gluon-private-wifi: add support for WPA3 encryption (#1876)
freifunk-gluon/gluon: docs/features/private-wlan.rst

It is possible to set up a private WLAN that bridges the WAN port and is separated from the mesh network.
Please note that you should not enable ``mesh_on_wan`` simultaneously.

> The private WiFi is by default encrypted via WPA2. On devices with enough flash and a supported radio,

Reviewer: Weird word order, better: "The private Wifi is encrypted via WPA2 by default."

> The private WiFi is encrypted via WPA2 by default. On devices with enough flash and a supported radio,

Reviewer 2: WiFi -> WLAN

Author: DOne

WPA3 or WPA2/WPA3 mixed-mode can be used instead of WPA2.

Additionally, IEEE 802.11w mangagement frame protection should be enforced for WPA2/WPA3, however this

> can lead to connectivity problems for older clients. In this case, mangagement frame protection can be

Reviewer 2: two times: "mangagement" -> "management"

Author: Fixed.

> set to optional or disabled completely using the config-mode. 

Reviewer: We usually refer to the web UI as "configuration wizard" or "advanced settings" instead of using the more technical term "config-mode" in the docs.

It is possible to set up a private WLAN that bridges the WAN port and is separated from the mesh network.
Please note that you should not enable ``mesh_on_wan`` simultaneously.

> The private WLAN is encrypted via WPA2 by default. On devices with enough flash and a supported radio,

Reviewer 3: ```suggestion
The private WLAN feature uses WPA2 encryption by default. On devices with enough flash and a supported radio,
```

Author: Altered this part according to @NeoRaider s previous feedback (which was overwritten by accident).

> The private WLAN is encrypted through WPA2 by default. On devices with enough flash and a supported radio,

Reviewer: Better: "through" -> "using"

WPA3 or WPA2/WPA3 mixed-mode can be used instead of WPA2. For this to work, the ``wireless-encryption-wpa3``
feature has to be added to the list of your supported features.

> Additionally, IEEE 802.11w management frame protection should be enforced for WPA2/WPA3, however this

Reviewer 3: ```suggestion
It is recommended to require IEEE 802.11w management frame protection for WPA2/WPA3 networks, however this
```

Author: done

can lead to connectivity problems for older clients. In this case, management frame protection can be
made optional or completely disabled in the advanced settings tab.

> 

Reviewer: Please add `:` after "docs" in the commit title.

> set to optional or disabled completely using advanced settings. 

Reviewer 3: ```suggestion
downgraded to be either optional or completely disabled using advanced settings. 
```

Author: Altered this part according to @NeoRaider s previous feedback (which was overwritten by accident).
